wordpress-mobile/gutenberg-mobile
# [Fix] Text formatting in caption

Fixes #2334 
gutenberg PR: https://github.com/WordPress/gutenberg/pull/22741

To test:
- Create a new image/video block with an image/video so the caption can be selected
- Select the caption
- Activate the bold, italic, or strikethrough formatting buttons
- Format button's state should change
- Begin typing text
- The correct format should be applied

PR submission checklist:
